frontend/src/components/phones/dashboard/Dashboard.tsx

```diff
@@ -1,12 +1,179 @@
+    relayNumberData.setForwardingState(
+      relayNumberData.data?.[0].number!,
+      !enableForwarding,
+      relayNumberData.data?.[0].id!
+    );
+  };
+
+  const copyPhoneNumber: MouseEventHandler<HTMLButtonElement> = () => {
+    // TODO: Find a way to not have to use a non-null assertion operator here
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    navigator.clipboard.writeText(relayNumberData?.data?.[0].number!);
+    setJustCopiedPhoneNumber(true);
+    setTimeout(() => setJustCopiedPhoneNumber(false), 1000);
+  };
+
+  const phoneStatistics = (
+    <div className={styles["phone-statistics-container"]}>
+      <div className={styles["phone-statistics"]}>
+        <p className={styles["phone-statistics-title"]}>12:04 min</p>
+        <p className={styles["phone-statistics-body"]}>
+          Remaining call minutes
+        </p>
+      </div>
+
+      <div className={styles["phone-statistics"]}>
+        <p className={styles["phone-statistics-title"]}>36</p>
+        <p className={styles["phone-statistics-body"]}>Remaining texts</p>
+      </div>
+
+      <div
+        className={`${styles["phone-statistics"]} ${
+          enableForwarding ? "" : styles["inactive-statistics"]
+        }`}
+      >
+        <p className={styles["phone-statistics-title"]}>7</p>
+        <p className={styles["phone-statistics-body"]}>
+          Calls and texts forwarded
+        </p>
+      </div>
+
+      <div
+        className={`${styles["phone-statistics"]} ${
+          enableForwarding ? styles["inactive-statistics"] : ""
+        }`}
+      >
+        <p className={styles["phone-statistics-title"]}>0</p>
+        <p className={styles["phone-statistics-body"]}>
+          Calls and texts blocked
+        </p>
+      </div>
+    </div>
+  );
+
+  const phoneControls = (
+    <div className={styles["phone-controls-container"]}>
+      <div className={styles["phone-controls"]}>
+        <button
+          onClick={toggleForwarding}
+          className={`${styles["base-button"]} ${
+            enableForwarding ? styles["active-button"] : ""
+          }`}
+        >
+          <ForwardIcon
+            alt="Forwarding All Messages"
+            className={styles["forward-icon"]}
+            width={15}
+            height={15}
+          />
+        </button>
+        <button
+          onClick={toggleForwarding}
+          className={`${styles["base-button"]} ${
+            enableForwarding ? "" : styles["active-button"]
+          }`}
+        >
+          <BlockIcon
+            alt="Blocking All Messages"
+            className={styles["block-icon"]}
+            width={15}
+            height={15}
+          />
+        </button>
+      </div>
+      <div className={styles["phone-controls-description"]}>
+        {enableForwarding ? (
+          <span>
+            Relay is currently forwarding all phone calls and SMS messages to
+            your true phone number.
```

Comment:
question (blocking): Are we trying to put all these in `pendingTranslations.ftl` so that, even though we're not *translating* everything for the first launch, we'll be *able* to translate them later?

Reply:
Yes - all the strings are going into pendingTranslations.